workbench/tests/renderer/investtab.test.js

```diff
@@ -223,9 +230,12 @@ describe('Sidebar Buttons', () => {
     const mockDialogData = { filePath: 'data.tgz' };
     ipcRenderer.invoke.mockResolvedValue(mockDialogData);
 
-    const { findByText, findByRole, getByRole } = renderInvestTab();
-
-    const saveButton = await findByText('Save datastack');
+    const { findByText, findByLabelText, findByRole, getByRole} = renderInvestTab();
+    const saveAsButton = await findByText('Save as...');
+    userEvent.click(saveAsButton);
+    const datastackOption = await findByLabelText((content, element) => content.startsWith('Parameters and data'));

```

Comment:
@davemfish I think this is why the test was flaky. I'd forgotten to update this line for this test case so it was actually testing the python script option, not the datastack option. The "archiving..." message would still appear briefly, but it wasn't getting the 100ms timeout defined above.

Reply:
@emlys Hmm, I do see that test is failing again actually, just on Mac. I'm playing with the timeout length in the mocked Promise and it seems like when it's too short, the alert has already updated to "saved", so the test never sees "archiving". On Windows I had to go down to 10ms to get the failure. But again, the slow Mac VMs might be main problem. Going up to 1000ms seems fine to me.

Reply:
ah okay. That test is reliable for me locally, so I'll just increase the timeout assuming it's a VM slowdown.